CentOS-6/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN yum -y install \
     glibc-devel.x86_64 \
     gmp-devel.x86_64 \
     libstdc++-devel.x86_64 \
+    tbb-devel \
```

Comment:
I have removed TBB from CentOS6, because it is the old version 2.2, and we only support versions >= 3.0.
See commit fdb16c8351a81835aada8f1d9ae36e2ebaa69b7b.